packages/loader/container-loader/src/container.ts

```diff
@@ -103,10 +102,37 @@ interface ILocalSequencedClient extends ISequencedClient {
     shouldHaveLeft?: boolean;
 }
 
+export interface IContainerLoadOptions {
+    /**
+     * Disables the Container from reconnecting if false, allows reconnect otherwise.
+     */
+    canReconnect?: boolean;
+    /**
+     * Client details provided in the override will be merged over the default client.
+     */
+    clientDetailsOverride?: IClientDetails;
+    /**
+     * Control whether to load from snapshot or ops.  See IParsedUrl for detailed information.
+     */
+    version?: string | null | undefined;
```

Comment:
i don't think this makes sense to specify at the loader layer as each container will have its own set of snapshot versions

Reply:
I'm not sure I follow -- this option is already specified at the loader layer via `LoaderHeader.version`?

Reply:
It's a "loader header" but it's provided by the request to the loader rather than part of some loader policy specified at creation, so does that still count as being specified at the loader layer?  Moving forward I think we probably want to use the request url route to specify this instead of the `LoaderHeader` anyway.  In which case the container could be responsible for extracting that instead of passing it explicitly here too.

Reply:
Chatted with Tony about this one - I think the key is that the version must not be persisted long-term in the `Loader` (since multiple calls to `loader.resolve`/`loader.request` may want different versions).  As long as its lifetime is a single `Container.load` call I think we're ok.